packages/playground/src/weblets/profile_manager.vue

```diff
@@ -873,7 +873,7 @@ function parseAcceptTermsLink(tempDiv: HTMLDivElement) {
 watch(openAcceptTerms, async () => {
   if (openAcceptTerms.value) {
     try {
-      const url = "https://library.threefold.me/info/legal/";
+      const url = "https://manual.grid.tf/knowledge_base/legal/terms_conditions_all3.html";
```

Comment:
it wont work; i think we should have markdown file to be able to parse it

Reply:
also looks like the host have cors error 
![image](https://github.com/user-attachments/assets/0d5d10a3-14b4-47ea-a426-1cb4a0757002)

Reply:
CORS error resolved, we may see the error as the current appends readme.md to the link, if you commented this part you will not face the cors error again, but the content will be shown as html file 
![image](https://github.com/user-attachments/assets/c00b67db-9351-4cd9-b20a-1084051ad733)